internal/learn/the-community/how-to-support.md

```diff
@@ -0,0 +1,109 @@
+---
+title: How to support the vision
+description: This page shows a list of ways to suport the vision
+---
+
+# How can you support the vision?
```

Comment:
Why would you write it in capital letters?

Reply:
Because those are our guidelines. Feel free to create a PR if you would like to change them :trollface: 
https://wiki.iota.org/learn/contribute-to-wiki/reference/style/formatting#headings